code/game/objects/effects/contraband.dm

```diff
@@ -453,6 +453,58 @@
 	desc = "DONK CO. BRAND MICROWAVABLE FOOD: MADE BY STARVING COLLEGE STUDENTS, FOR STARVING COLLEGE STUDENTS."
 	icon_state = "poster51"
 
+/obj/structure/sign/poster/contraband/donk_co/examine_more(mob/user)
+	var/list/msg = list(span_notice("<i>You browse some of the poster's information...</i>"))
+	msg += "\t[span_info("DONK CO. BRAND DONK POCKETS: IRRESISTABLY DONK!")]"
+	msg += "\t[span_info("AVAILABLE IN OVER 200 DONKTASTIC FLAVOURS: TRY CLASSIC MEAT, HOT AND SPICY, NEW YORK PEPPERONI PIZZA, BREAKFAST SAUSAGE AND EGG, PHILADELPHIA CHEESESTEAK, HAMBURGER DONK-A-RONI, CHEESE-O-RAMA, AND MANY MORE!")]"
+	msg += "\t[span_info("AVAILABLE FROM ALL GOOD RETAILERS, AND MANY BAD ONES TOO!")]"
+	return msg
+
+/obj/structure/sign/poster/contraband/cybersun_six_hundred
+	name = "Saibāsan: 600 Years Commemorative Poster"
+	desc = "An artistic poster commemorating 600 years of continual business for Cybersun Industries."
+	icon_state = "poster52"
```

Comment:
The moth posters at the bottom allude to this (despite my dislike of historical comments)--can we not make these all just numbers going up and instead just name them exactly what they are? Like just naming it `poster_cybersun_six_hundred` for instance?

Reply:
this battle of poster[number] versus real names is a battle for the ages the likes the world has not seen since the Boshin war

I can do that though, and since I'm touching the posters, I could go through and do the same for the rest of the posters in the game, if you fancy?

Reply:
Yeah if you want